packages/components/organisms/f-checkout/stories/Checkout.stories.js

```diff
@@ -49,21 +51,34 @@ CheckoutMock.setupCheckoutMethod(updateCheckoutServerErrorUrl);
 CheckoutMock.setupCheckoutMethod(getAddressUrl);
 CheckoutMock.setupCheckoutMethod(placeOrderUrl);
 CheckoutMock.setupCheckoutMethod(placeOrderDuplicateUrl);
+CheckoutMock.setupCheckoutMethod(accessForbiddenErrorUrl);
+CheckoutMock.setupCheckoutMethod(getCheckoutErrorUrl);
 CheckoutMock.setupCheckoutMethod(getGeoLocationUrl);
 CheckoutMock.passThroughAny();
 
 const checkoutIssues = 'Checkout Issues (Response from server but order not fulfillable)';
 const checkoutServerError = 'Checkout Error (Response from server is an error)';
 const placeOrderError = 'Place Order Duplicate Error (Response from server is an error)';
+const accessForbiddenError = 'Access Forbidden Get Checkout Error (Response from server is an error)';
+const defaultGetCheckoutError = 'Any other Get Checkout Error (Response from server is an error)';
 const ISSUES = 'ISSUES';
 const SERVER = 'SERVER';
+const accessForbiddenErrorCode = '403';
+const defaultGetCheckoutErrorCode = 'default';
```

Comment:
This looks a bit weird, since it says `ErrorCode` but then it says `default`.

Reply:
It is used for all errors other than 403 so I came up with 'default'. Can you suggest something more logical for it?

Reply:
A standard `500` error should be "default enough" in my opinion, hehe.